### PR TITLE
feat: total_expenses in trip serializer

### PR DESCRIPTION
**Pull request by IgorrMagalhaess**
## Type of change
- [x] New feature
- [ ] Bug Fix
- [x] Test
- [ ] Refactor/Style

## Description
Relevant User Story(s):
*Brief summary of the change(s) included in this pull request*
Add total_expense to JSON return of trips endpoints
This description closes #

## Check the correct boxes
- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes
- [ ] No Tests have been changed
- [x] Some Tests have been changed
- [ ] All of the Tests have been changed

*Please describe what in the world happened if tests are broken*

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

## Fun fact or favorite .gif:
Canada is south of Detroit (just look at a map). 🤯

### CODE REVIEW

File: app/serializers/trip_serializer.rb [hunk 1]
```diff
@@ -1,4 +1,11 @@
...
   include JSONAPI::Serializer
   attributes :name, :location, :start_date, :end_date, :status, :total_budget
+
+  has_many :daily_itineraries
+  has_many :activities, through: :daily_itineraries
+
+  attribute :total_expenses do |object|
+    object.activities.sum(&:expenses) || 0
+  end
 end
```
Comment (yainporter) [inline app/serializers/trip_serializer.rb:11]: Nice!

**Review by yainporter — APPROVED** [review]
Looks great, Igor, thanks for getting that set up!

**Comment by yainporter** [comment]
Also, thanks for fixing the spacing!!! :D

**Review by hoadam — APPROVED** [review]
Great work! Thank you

**Pull request merged**